### DISCUSSION TITLE
Would a PS/2 keyboard adapter/cable be possible?

### DISCUSSION THREAD

**Discussion by gbraad** [darthcloud/BlueRetro#1020] [Q&A]
I have several FPGA-based boards, and they all come with a PS/2 port for the keyboard. Is there a way to use Blueretro to connect a BT-based keyboard and use PS/2 out? I read that the Dreamcast cable for the Core did something similar.

**Comment by darthcloud** [comment]
Yes it's something I got on my mind for a while. I dont know when tho.
